packages/logo/README.md

```diff
@@ -1,6 +1,8 @@
 # [`@fremtind/jkl-logo`](https://fremtind.github.io/jokul/components/logo/)
 
-Logoen til Fremtind er navnet skrevet ut. Logosymbolet er en F som tipper fremover og former en tind. Logo og logosymbolet kan animeres slik at vi kan veksle mellom de. Vi bruker ikke logo og logosymbolet sammen eller ved siden av hverandre.
+# Logo, logosymbol og stempel
+Logo, logosymbolet og logostemplene våre er merkevareelementer vi kan bruke i Fremtinds løsninger eller i andre løsninger vi er en del av.
 
-### Logostemplene
-Logostemplene er ment for å synliggjøre Fremtind som tilstedeværende eller som en ingrediens i budskapet som formidles. De brukes primært på distributørflater for å synliggjøre Fremtinds rolle.
+For bruk av logo og logosymbol se prinsippene for bruk. Disse kan animeres slik at det kan veksles mellom de men skal ikke stå synlige sammen.
```

Comment:
_Forslag:_ 
For bruk av logo og logosymbol se prinsippene for bruk (_tekstlenke_). 
Logoene skal ikke stå synlige sammen men kan skifte ved hjelp av animasjon.